packages/react-cards/src/components/Card/Card.view.tsx

```diff
@@ -3,21 +3,29 @@ import { withSlots, getSlots } from '@uifabric/foundation';
 import { getNativeProps, htmlElementProperties } from '@uifabric/utilities';
 import { Stack } from 'office-ui-fabric-react';
 
-import { ICardComponent, ICardProps, ICardSlots } from './Card.types';
+import { ICardComponent, ICardProps, ICardSlots, ICardTokens } from './Card.types';
 
 export const CardView: ICardComponent['view'] = props => {
   const Slots = getSlots<ICardProps, ICardSlots>(props, {
     root: 'div',
     stack: Stack
   });
 
-  const { compact, gap, ...rest } = props;
+  const { tokens, compact, ...rest } = props;
+  const cardTokens = tokens && typeof tokens !== 'function' ? (tokens as ICardTokens) : undefined;
+  const { childrenGap } = cardTokens ? cardTokens : { childrenGap: 12 };
```

Comment:
I think this can be `const { childrenGap = 12 } = cardTokens;`

Reply:
I think so too.. would be easier to parse